sonar-project.properties

```
@@ -0,0 +1,12 @@
+sonar.organization = luno
+sonar.projectKey = luno_weld
+sonar.projectName = weld
+sonar.links.scm = https://github.com/luno/weld
+
+sonar.sources = .
+sonar.exclusions=**/*_test.go, **/*pb.go, _examples/**/*, adapters/reflexstreamer/**/*, adapters/kafkastreamer/**/*, adapters/sqlstore/**/*, adapters/sqltimeout/**/*
```

Some of these paths are specifc to workflow, so we don't need them here! 😄 
```suggestion
sonar.exclusions=**/*_test.go, **/*pb.go
```